build.gradle

```diff
@@ -35,12 +35,13 @@ shadowJar {
 }
 
 dependencies {
-    compile  "org.embulk:embulk-core:0.9.20"
-    provided "org.embulk:embulk-core:0.9.20"
+    compile  "org.embulk:embulk-core:0.9.23"
+    provided "org.embulk:embulk-core:0.9.23"
```

Comment:
org.embulk.deps.buffer.PooledBufferAllocator has been provided since 0.9.22.
https://github.com/embulk/embulk/commit/9d93f9e6e0b7ad38caae2f8529a9861c95491e76